### PR TITLE
Adding grpc library and generated code from grpc-tools

### PR DESCRIPTION
**Pull request by noiarek**
**What changed?**
Adding grpc library

**Why?**
To migrate tchannel connection to grpc

**How did you test it?**
Next PR test this

**Potential risks**
Nothing

**Release notes**
No

**Documentation Changes**
No

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+generate-grpc-code:
```
Comment (longquanzheng) [inline Makefile:1]: How about add some instructions on how to run the command to re-generate or refresh those code?

**Review by longquanzheng — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (noiarek) [reply Makefile:1]: Good call! 

Should we consider adding it on `CONTRIBUTING.md` instead? or what do you think

Comment (longquanzheng) [reply Makefile:1]: Yeah I think that sounds great

Comment (noiarek) [reply Makefile:1]: Done. Fully inspired from the cadence server one :)

**Pull request merged**